mods/lm-close-windbg-file-changed.wh.cpp

```diff
@@ -0,0 +1,77 @@
+// ==WindhawkMod==
+// @id           lm-close-windbg-file-changed
+// @name         Close WinDbg's file changed dialog
+// @description  Automatically rejects (or accepts) the dialog from WinDbg telling that a file is changed
+// @version      1.0
+// @author       Mark Jansen
+// @github       https://github.com/learn-more
+// @twitter      https://twitter.com/learn_more
+// @include      windbg.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Close WinDbg's file changed dialog
+This mod automatically dismisses the popup from WinDbg notifying about a file change.
+When automating WinDbg, this dialog blocks progress.

```

Comment:
How about adding a screenshot? It should be as easy as uploading it to https://imgur.com/ and adding it here like this:
https://github.com/ramensoftware/windhawk-mods/blob/6c2b4e0ee6af475c451c1ae866e54b312a328195/mods/notepad-dark-mode.wh.cpp#L24

Reply:
Screenshot added.